packages/core/src/dynamoDb/utils/dynamodbUtils.ts

```diff
@@ -0,0 +1,77 @@
+/*!
```

Comment:
can just rename this file to just dynamodb.ts instead of dynamodbutils, since it's already in the utils folder. Then it would be:
```
dynamoDb/utils/dynamodb.ts
```

Reply:
renamed